### DISCUSSION TITLE
"RSVP not yet allowed text" not showing for multibooking form

### DISCUSSION THREAD

**Discussion by refugeindhamma** [liedekef/events-made-easy#126] [Q&A] [answered]
Hi Franky,

I set the Recurrent event to allow RSVP 30 days before the event starts in June 2023.

I saw on the "RSVP" settings there is a message for "RSVP not yet allowed text". However, on the front end, the Multibooking Body is hidden while the header and footer are shown. How do I make the message appear and hide the entire multibooking form?

**Comment by liedekef** [comment, marked as answer]
In case of multibooking, the 'RSVP not yet allowed' text is not shown (because other events can show and then it would be mixup of seats to book and 'not yet allowed' messages).
The header/footer templates (if used) are used for all events in your multibooking form, so it is not possible to test the rsvp start date of 1 specific event.

**Reply by refugeindhamma** [reply]
Thanks for the clarification.

Since the body template is hidden before RSVP starts, can you make the header & footer be hidden as well?

**Reply by liedekef** [reply]
the body template is NOT hidden, but if you don't have any events to show, it is just empty. Also, like I said: the header/footer template have no notion of a specific event, so there is nothing to show/hide because there's no logic to apply.
